Updates to emails, spatial and others (#138)
NCEAS/datateam-training: workflows/pi_correspondence/email_templates/A_initial_email_template.Rmd

### Initial email template

> Hello [NAME OF REQUESTOR],
> Thank you for your recent submission to the NSF Arctic Data Center!
> 
> From my preliminary examination of your submission I have noticed a few items that I would like to bring to your attention. We are here to help you publish your submission, but your continued assistance is needed to do so. See comments below:
> 
> [COMMENTS HERE]
>

> > After we recieve your responses, we can make the edits on your behalf.

Reviewer: ```suggestion
> After we receive your responses, we can make the edits on your behalf, or you are welcome to make them yourself using our user interface.
```

> I noticed that your data package has incomplete attribute/variable information. [SPECIFY WHAT YOU WANT ADDED] You can do so by navigating to your dataset [INSERT URL HERE], clicking the green "Edit" button, clicking the "Describe" button located to the right of each of your files at the top of the submission page, and in the window that pops up click over to the "Attributes" tab. Please see the guidance below for what attribute/variable information should contain.

Checklist for the datateam in reviewing attributes (NetCDF, CSV, shapefiles, or any other tabular datasets):
> - A name (often the column or row header in the file).
> - A complete definition.
> - Any missing value codes along with explanations for those codes.
> - For all numeric data, unit information is needed.
> - For all date-time data, a date-time format is needed (e.g. "DD-MM-YYYY").
> - For text data, full descriptions for all patterns or code/definition pairs are needed if the text is constrained to a list of patterns or codes.

> Could you describe ____? 
> Please define “XYZ”, including the unit of measure. 
> What are the units of measurement for the columns labeled “ABC” and “XYZ”?

Missing value codes

> > What do the missing values in your measurements represent? Something such as a short description (instrument failure, site not found, etc.) will suffice.

Reviewer: ```suggestion
> What do the missing values in your measurements represent? A short description of the reason why the values are missing (instrument failure, site not found, etc.) will suffice.
```

> > We noticed that the data files contain [blank cells - replace with missing values found]. What do these represent? 

Reviewer: I'm not totally sure what the revision means here. Is replace with missing values found an instruction for the email sender?

#### Funding

All NSF funded datasets need a funding number. Non-NSF funded datasets might not have funding numbers, depending on the funding organization.

> > We noticed that your dataset does not appear to contain a funding number. The field accepts NSF numbers as well as other numbers by different organizations.

Reviewer: ```suggestion
> We noticed that your dataset does not appear to contain a funding number. The field accepts NSF funding numbers as well as other numbers by different organizations.
```

#### Methods

Submissions should:

> - provide instrument names

Reviewer: ```suggestion
- provide instrument names (if applicable)
```
